recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ test:
 
 about:
   home: http://pygithub.github.io/PyGithub/v1/
-  license: GNU Library or Lesser General Public License (LGPL)
+  license: GPL-3.0
```

Comment:
So, this is incorrect as it is not under GPL-3. It is instead under LGPL-3. Please see PR ( https://github.com/conda-forge/pygithub-feedstock/pull/10 ) for a correction.